lib/osut/utils.rb

```diff
@@ -2945,9 +2999,16 @@ def getLineIntersection(s1 = [], s2 = [])
     a1b2  = b2 - a1
     xa1b1 = a.cross(a1b1)
     xa1b2 = a.cross(a1b2)
+    xa1b1.normalize
+    xa1b2.normalize
+    xab.normalize
```

Comment:
Quite the oversight not to _normalize_, especially when dealing with one line segment possibly orders of magnitude longer than the other.